app/web_conn_test.go

```diff
@@ -254,10 +254,13 @@ func TestWebConnDrainDeadQueue(t *testing.T) {
 			i := seqNum
 			for err == nil {
 				_, buf, err = conn.ReadMessage()
-				ev := model.WebSocketEventFromJson(bytes.NewReader(buf))
-				require.LessOrEqual(t, int(i), limit)
-				assert.Equal(t, i, ev.GetSequence())
-				i++
+				if err != nil && len(buf) > 0 {
```

Comment:
Added this to make the test pass. Let me know if we should instead accept empty inputs.

Reply:
Fine with me.